src/main/msp/msp.c

```diff
@@ -3121,10 +3126,15 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
             sbufReadU16(src);
 #endif
 #if defined(USE_RPM_FILTER)
-            rpmFilterConfigMutable()->rpm_filter_harmonics = sbufReadU8(src);
+            for (int i = 0; i < RPM_FILTER_HARMONICS_MAX; i++) {
```

Comment:
Changing RPM_FILTER_HARMONICS_MAX needs to be reflected in configurator - it will lcause problems.
Prefix it with  arrays size (in place of old rpm_filter_harmonics)